### PR TITLE
Firefox 68 released

### PR DESCRIPTION
**Pull request by afmenez**
Firefox 68 will be released tomorrow.

### CODE REVIEW

**Review by chicoxyzzy — COMMENTED** [review]
Numeric Separators and Promise.allSettled are not available by default in Firefox 68 and Firefox 69

File: environments.json [hunk 1]
```diff
@@ -1278,21 +1278,28 @@
     "obsolete": false
   },
   "firefox68": {
-    "full": "Firefox 68 Beta",
+    "full": "Firefox 68",
```
Comment (chicoxyzzy) [inline environments.json]:
```suggestion
    "full": "Firefox 68 ESR",
```

File: environments.json [hunk 2]
```diff
@@ -1278,21 +1278,28 @@
...
     "family": "SpiderMonkey",
-    "short": "FF 68 Beta",
+    "short": "FF 68",
```
Comment (chicoxyzzy) [inline environments.json]:
```suggestion
    "short": "FF 68 ESR",
```

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

File: environments.json (hunk 1 above)
Comment (afmenez) [reply environments.json]: 68 ESR wasn't released yet.

File: environments.json (hunk 2 above)
Comment (afmenez) [reply environments.json]: 68 ESR wasn't released yet.

**Comment by afmenez** [comment]
You are correct, both features  seem to be "beta and nightly only" for now.  I'll update the patch later today.

File: environments.json (hunk 2 above)
Comment (chicoxyzzy) [reply environments.json]: https://www.mozilla.org/en-US/firefox/68.0esr/releasenotes/

**Comment by chicoxyzzy** [comment]
> both features seem to be "beta and nightly only" for now

Both are disabled in Beta and Developer Edition and enabled in Nightly only

**Comment by afmenez** [comment]
Fixed all the wrong results (found another one on the non-standard page).  The 68 ESR builds are not released yet, we can update this later (in a separate PR).

**Comment by chicoxyzzy** [comment]
ESR was released see https://github.com/kangax/compat-table/pull/1481#discussion_r301681697

**Comment by afmenez** [comment]
Yep, the release notes for all versions is usually available in advance, but the installer it is not available on the [main download page](https://www.mozilla.org/en-US/firefox/organizations/) yet.
Also, current 60 ESR are not being upgraded yet.
The ESR is  usually released only after the main release is already "unthrottled" and is considered trusted enough.  It takes a few days.

**Comment by chicoxyzzy** [comment]
I don't see any reasons why we shouldn't add ESR now.

> Also, current 60 ESR are not being upgraded yet.

It will get updates until 70 / 68.2 ESR will be released according to calendar https://wiki.mozilla.org/Release_Management/Calendar

**Comment by chicoxyzzy** [comment]
Also you can download 68 ESR here: https://www.mozilla.org/en-US/firefox/organizations/all/?q=English%20(US)

File: environments.json (hunk 2 above)
Comment (afmenez) [reply environments.json]: Done.

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**